workflow-templates/release-please.yml

```diff
@@ -0,0 +1,53 @@
+name: release-please
+
+# Control when the action will run
+on:
+  # Triggers the workflow on push events except for the main branch
+  push:
+    branches: [ $default-branch ]
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      # uses the GitHub action from google to automate releases.
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: node
+          # You should change this to the repo name
+          package-name: USE_YOUR_REPO_NAME
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"docs","section":"Documentation","hidden":false}]'
+
+      # checkout the code to get the latest version
+      - uses: actions/checkout@v3
+
+      # in case a release was created, this will create or update the major and minor tags
+      # so you can refer to `v1` or `v1.1` and get the latest version corresponding to that prefix.
+      - name: Tag major and minor versions
+        uses: jacobsvante/tag-major-minor-action@v0.1
+        if: ${{ steps.release.outputs.release_created }}
+        with:
+          major: ${{ steps.release.outputs.major }}
+          minor: ${{ steps.release.outputs.minor }}
+
+# -------------------------------------------------------------------------
+# This section is useful only if you plan on publishing your package to NPM
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+        if: ${{ steps.release.outputs.release_created }}
+
+      - name: Ensure clean install
+        run: yarn install --immutable
+        if: ${{ steps.release.outputs.release_created }}
+
+      - name: Publish to NPM
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        if: ${{ steps.release.outputs.release_created }}
```

Comment:
Should we disable this part by default? Or have inputs to enable/disable?

Reply:
Should we have a workflow in graasp-deploy, that will be called in this template?

Reply:
@spaenleh, any reply to these comments by Kim?

Reply:
@juancarlosfarah I did not have time to work on this lately, will get back to it when i have time, but yes we should make this optional and refactor it as re-usable !